internal/config/config.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/viper"
+)
+
+// DB contains database and migration settings
+type DB struct {
+	ConnString       string        `mapstructure:"conn_string"`
+	MaxOpenConns     int           `mapstructure:"max_open_conns"`
+	ConnMaxLifetime  time.Duration `mapstructure:"conn_max_lifetime"`
+	MigrationDirPath string        `mapstructure:"migration_dir_path"`
+	MigrationTable   string        `mapstructure:"migration_table"`
+}
+
+// API contains api settings
+type API struct {
+	Bind string `mapstructure:"bind"`
+}
+
+// AppConfig is a container for application config
+type AppConfig struct {
+	API *API `mapstructure:"api"`
+	Db  *DB  `mapstructure:"db"`
+}
+
+// GetAppConfig returns *Config
+func GetAppConfig() (*AppConfig, error) {
+	viper.SetConfigName("config")
```

Comment:
Please add an explanation (comment/ readme) that this config file is hardcoded

Reply:
added both inline comment and README description